rmgpy/data/kinetics/groups.py

```diff
@@ -196,6 +196,8 @@ def estimateKineticsUsingGroupAdditivity(self, template, referenceKinetics, dege
             kinetics.kdata.value_si *= degeneracy
         elif isinstance(kinetics, Arrhenius):
             kinetics.A.value_si *= degeneracy
+        elif isinstance(kinetics, ArrheniusEP):
+            kinetics.A.value_si *= degeneracy
```

Comment:
getting rid of the `elif` statements with `kinetics.changeRate(degeneracy)` might be cleaner.